modules/rewrite/classes/parsers/isapirewrite/IsapiRewriteUrlGenerator.class

```diff
@@ -29,6 +29,8 @@ GalleryCoreApi::requireOnce('modules/rewrite/classes/RewriteUrlGenerator.class')
  * @version $Revision: 17580 $
  */
 class IsapiRewriteUrlGenerator extends RewriteUrlGenerator {
+    /* deprecated dynamic properties in php 8.2 */
+    var $_rootItemId;
```

Comment:
I don't think this is needed

Reply:
The variable is inherited from 'class GalleryUrlGenerator' so you are correct, the declaration is not necessary here. Do you want me to revoke the pull request, correct and send a new pull request? I have another pull request coming up with a few PostgreSQL specific patches so it might be easier if I just include a correction in that coming pull request

Reply:
Fixing it in the upcoming pull request is fine.

For future reference, I *think* that it's easier to make a separate branch when you have a PR to do - that way it's easier to make changes as needed without interfering with other work. I'm not real familiar with the GitHub workflow, though, so I might be wrong :-) Anyway, this isn't a big deal so fixing it later is fine.